### PR TITLE
chore(eslint): add no unused var rule + fix related lint errors

### PR DESCRIPTION
**Pull request by Boyadjie**
## 🤔 Why do you want to make those changes?

Because we don't want to let unused imports pass on the app anymore

## 🧑‍🔬 How did you make them?

I've added the rule no-unused-vars to the `.eslintrc` file

## 🧪 How to check them?

- read the code
- read checks and logs

### CODE REVIEW

File: src/app/utils/formatUrlWithQuery.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ export type QueryParams = {
 
 export const formatUrlWithQuery = (queryParams: QueryParams, path: string) => {
 	const queryString = Object.entries(queryParams)
-		.filter(([key, value]) => value)
+		.filter(([value]) => value)
```
Comment (Slashgear) [inline src/app/utils/formatUrlWithQuery.ts:7]: Hum.... Are we sure of this ?

**Review by Slashgear — COMMENTED** [review]
Otherwise nice

File: src/app/utils/formatUrlWithQuery.ts (hunk 1 above)
Comment (Boyadjie) [reply src/app/utils/formatUrlWithQuery.ts:7]: we don't need the `[ ]`around value but except that it looks good. What's bothering you ?

**Review by CruuzAzul — APPROVED** [review]

**Comment by CruuzAzul** [comment]
⚠️ I think you have elements in your PR that should not be there, for example the logo

File: .eslintrc [hunk 2]
```diff
@@ -1,10 +1,19 @@
...
-		"no-console": ["error", {"allow": ["warn", "error"]}]
+		"no-console": [
+			"error",
+			{
+				"allow": ["warn", "error"]
+			}
+		],
+		"@typescript-eslint/no-unused-vars": "error",
+		"@typescript-eslint/no-empty-function": "off",
+		"no-unused-vars": "off"
```
[the remark this answers is not in the document]
Comment (CruuzAzul) [reply .eslintrc:17]: Ok, but if we have a variable that is not used we don't get the error ?

**Pull request merged**